src/ucs/sys/string.c

```diff
@@ -199,6 +200,17 @@ ucs_status_t ucs_str_to_memunits(const char *buf, void *dest)
     return UCS_OK;
 }
 
+char *ucs_dirname(char *path, int num_layers)
+{
+    while (num_layers--) {
```

Comment:
> 0

Reply:
@yosefe Should not have problem when num_layers==0 since "--" is after num_layers. When num_layers==0, won't go into while loop, will only return path.

Reply:
sorry, github re-formatted my comment
i've meant:
`which (num_layers-- > 0)`

Reply:
@yosefe Fixed.